src/ViewRoom/ViewRoom.js

```diff
@@ -46,66 +47,66 @@ class ViewRoom extends Component {
           <div className="view-room-contents-right">
             <div className="view-room-contents-right-inner">
               <div className="textbox">
-                <div className="label">
+                <div className="room-id-label">
                   <p>Room ID:</p>
                 </div>
-                <textarea className="textarea" rows="2">
+                <textarea className="room-id-textarea" rows="2">
                 </textarea>
               </div>
               <div className="textbox">
-                <div className="label">
+                <div className="room-name-label">
                   <p>Room Name:</p>
                 </div>
-                <textarea className="textarea" rows="2">
+                <textarea className="room-name-textarea" rows="2">
```

Comment:
Removing `className="label"` and `className="textarea"` has removed the stylings from these elements.
Maybe rather than replacing the classNames use the id property as a selector instead.
So for example, here it would be `<textarea className="textarea" id="room-name-textarea">`.
Then just change the test selectors from `.room-name-textarea` format to `#room-name-textarea`.